lacework/resource_lacework_integration_ghcr.go

```diff
@@ -0,0 +1,272 @@
+package lacework
+
+import (
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkIntegrationGhcr() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkIntegrationGhcrCreate,
+		Read:   resourceLaceworkIntegrationGhcrRead,
+		Update: resourceLaceworkIntegrationGhcrUpdate,
+		Delete: resourceLaceworkIntegrationGhcrDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkIntegration,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"registry_notifications": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"credentials": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"username": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"password": {
+							Type:      schema.TypeString,
+							Sensitive: true,
+							Required:  true,
+						},
+						"ssl": {
```

Comment:
I think when it is a username and password, we leave these inputs at the top level, that is, we don't wrap them into a `credentials {}` block, at least that is what I see for Docker Hub. If we think this is not right, we could leave it as is but at least leave `ssl` outside since that setting has nothing to do with Credentials.